src/utils/ngcc-jest-processor.ts

```diff
@@ -55,10 +71,17 @@ export const runNgccJestProcessor = (tsconfigPath: string | undefined): void =>
       throw new Error(`${errorMessage} NGCC failed ${errorMessage ? ', see above' : ''}.`);
     }
   } else {
-    console.log(
-      `Warning: Could not locate '@angular/compiler-cli' to run 'ngcc' automatically.` +
-        `Please make sure you are running 'ngcc-jest-processor.js' from root level of your project.` +
-        `'ngcc' must be run before running Jest`,
-    );
+    const ngCompilerCliVersion = findAngularCompilerCliVersion();
+    const [ngMajorVersion] = ngCompilerCliVersion.split('.');
+
+    if (parseInt(ngMajorVersion, 10) < 16) {
+      console.log(
+        `Warning: Could not locate '@angular/compiler-cli' to run 'ngcc' automatically.` +
+          `Please make sure you are running 'ngcc-jest-processor.js' from root level of your project.` +
+          `'ngcc' must be run before running Jest`,
+      );
+    } else {
+      console.log(`@angular/compiler-cli@${ngCompilerCliVersion} detected. Skipping 'ngcc'`);
```

Comment:
Not sure about this message.

Another optional is to keep the original warning for NG15 and lower and don't log anything for NG16 or higher

Reply:
I think we can add extra message to this logging to tell users that they can remove `jest-preset-angular/global-setup` from their Jest config to avoid having this logging

Reply:
done 👍